src/main/scala/nl/biopet/utils/tool/ToolCommand.scala

```diff
@@ -203,6 +206,14 @@ trait ToolCommand[Args] extends Logging {
     }
     configFile.write(config)
     configFile.close()
+
+    Documentation.htmlRedirector(
```

Comment:
Can you prefix this with something like this: 
```scala
if (!version.endWith("-SNAPSHOT"))
```

Maybe even an else where the redirect will be placed at ./latest instead of the root